scripts/test/lint.py

```diff
@@ -86,6 +86,26 @@ def run_cmd(cmd):
     return out
 
 
+class IncludesDescriptionLength(Lint):
+    max_length = 175
+    min_length = 10
+    name = "the description length is invalid"
+    recommendation = "See the VM-Packages Wiki for conventions and package structure related to the description"
+
+    def check(self, path):
+        dom = minidom.parse(str(path))
+        metadata = dom.getElementsByTagName("metadata")[0]
+        description = metadata.getElementsByTagName("description")[0]
+        text = description.firstChild.data
+        length = len(text)
+        if not (self.min_length <= length <= self.max_length):
```

Comment:
this can be written using `range`, making it more readable. Remove the `length` variable or rename it to `description_length` so that it is clear what it is.

```suggestion

        if len(description) not in range(self.min_length, self.max_length):
```